File: Napakalaki/src/NapakalakiGame/DeathBadConsequence.java
/**
 * DeathBadConsequence.java
 */

package NapakalakiGame;

/**
 * Representa un mal rollo de muerte.
 */
public class DeathBadConsequence extends NumericBadConsequence {
    
    public DeathBadConsequence(String t) {
        super(t, Player.MAXLEVEL, MAXTREASURES, MAXTREASURES);
    }
    
}

File: Napakalaki/lib/death_bad_consequence.rb
# encoding: utf-8

require_relative 'player'
require_relative 'numeric_bad_consequence'

module NapakalakiGame

  # class BadConsequence: representa el mal rollo de un monstruo
  class DeathBadConsequence < NumericBadConsequence
    def initialize(t)
      super(t, Player.getMaxLevel, @@MAXTREASURES, @@MAXTREASURES)
    end
  end
end
